tools/run-dev

```diff
@@ -204,6 +204,7 @@ def server_processes() -> list[list[str]]:
             "--quiet",
         ],
         ["./manage.py", "deliver_scheduled_messages", "--automated"],
+        ["./manage.py", "clear_user_status"],
```

Comment:
Hmm, is there a way we can avoid adding another process in the development environment here? We're trying to keep resources minimal in the development environment.

@alexmv FYI in case you have ideas for what we should do. Options I see include:
- Make `manage.py process_queue` incorporate these non-queue workers.
- Merging this with `deliver_scheduled_messages` at least.

(There's kinda a similar goal of avoiding an extra process for low-RAM self-hosted systems)